lib/inspec/utils/telemetry/collector.rb

```diff
@@ -21,7 +27,11 @@ def add_data_series(data_series)
     # Always true until we add configuration parsing.
     # @return [True, False]
     def telemetry_enabled?
-      @enabled
+      if @config.telemetry_enabled?.nil?
+        @enabled = true
+      else
+        @enabled = @config.telemetry_enabled?
+      end
     end
```

Comment:
I know this is a bit weird if `@config` is nil and thus the `telemetry_enabled?` method won't exist. This might be more problematic if InSpec is used as a 'library', but if the cli/config is loaded properly we should be good. Suggestions?

Reply:
I'm not sure I understand your comment here. If `@config` is nil, that first line will fail calling `telemetry_enabled?` (unless we're doing hinky things on `NilClass`) and will raise a `NoMethodError`.

Reply:
Inspec::Config.cached will still work if `inspec` is used as a library. Since that is the default, @config would only be nil if someone were actively trying to break it :-)

Reply:
My only issue is that `Inspec::Config.cached` doesn't work 'right' if you are testing the collector class. This is why I'm passing in a mock config object.